source/ArduinoCommon.h

```diff
@@ -25,9 +25,11 @@ const UCHAR MSBFIRST = 0x01;
 
 const UCHAR NUM_ARDUINO_PINS = 20;
 
-const UCHAR CHANGE = 0x01;
+const UCHAR RISING = 0x01;
```

Comment:
Yes, we define them differently.  I believe wiring_constants.h contains Arduino hardware specific values.